component/Cards/card.css

```diff
@@ -0,0 +1,114 @@
+
+.parent-positioning img{
+    width:100%;
+    height:21rem;
+    object-fit:cover;
+}
+.vertical-cards{
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 3rem;
+}   
+.card-container{
+    display:flex;
+    flex-wrap:wrap;
+    max-width:27rem;
+    margin:0 auto;
+    margin-top:6rem;
+    background-color:var(--white-color);
+    box-shadow: 0px 3px 10px lightgray ;
+}
+.inside-container{
+    padding:8px 10px 3px 10px;
+    line-height: 2.5rem;
+}
+.card-content{
+    font-size: 1.6rem;
+    margin-top:1.5rem;
+}
+.card-button{
+    margin: 2rem 2rem 1rem 0;
+}
+.icon-size{
+    font-size: 2rem;
+}
+.parent-positioning{
+    position: relative;
+}
+.icon-overlay{
+    position: absolute;
+    top:1rem;
+    right:5px;
+    background-color: var(--white-color);
+    height:2.5rem;
+    width:2.7rem;
+    border-radius: 50%;
+    padding:3px 0 0 3.5px;
+}
+.icon-overlay:hover{
+    cursor: pointer;
+    color:var(--alert-error-color);
+}
+.cost{
+    font-weight: bold;
+    font-size: 1.8rem;
+}
```

Comment:
Everything in code looks good just one suggestion one word cost confuses a bit you could have used something like text-cost, card-price, I am not that great at naming but point is later it may become handy to understand. For now all good I don't think its necessary to change but maybe used something along with it so thats easily understood. Rest all is awesome.

Reply:
thanks for the feedback what you are saying is right i will make the changes